commit 23d04334b0a4c2fc24131e01ecf153c02c90a656
Author: Trent Mick <trentm@gmail.com>
Date:   2019-06-27T14:52:59-07:00 (3 months ago)
    
    TRITON-1779 sapi mode.test.js creates cpu_cap-less VMs
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
    Approved by: Todd Whiteman <todd.whiteman@joyent.com>

diff --git a/test/common.js b/test/common.js
index a6cfd0d..37dc792 100644
--- a/test/common.js
+++ b/test/common.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2015, Joyent, Inc.
+ * Copyright 2019 Joyent, Inc.
  */
 
 /*
@@ -53,6 +53,7 @@ function createService(app_uuid, uuid, cb) {
     var opts = {};
     opts.params = {};
     opts.params.ram = 256;
+    opts.params.cpu_cap = 100;
     opts.params.networks = [ 'admin' ];
     assert.string(process.env.SAPI_TEST_IMAGE_UUID,
         'process.env.SAPI_TEST_IMAGE_UUID');
